client/src/js/constants/bhConstants.js

```diff
@@ -10,6 +10,9 @@ function constantConfig() {
   var JOURNAL_UTIL_HEIGHT = '150px';
 
   return {
+    actions : {
+      CAN_EDIT_ROLES : 1,
```

Comment:
Why do you have actions both here and in the database?  It seems like they should be in one place, not in two.

Reply:
your right but to my mind I thought that using action's id in the controllers is not a good thing that is why I created constants instead